# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/TextureClientPool.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/layers/client/TextureClientPool.cpp
--- a/gfx/layers/client/TextureClientPool.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/layers/client/TextureClientPool.cpp	Wed May 22 22:27:37 2019 +0000
@@ -9,8 +9,7 @@
 #include "mozilla/layers/CompositableForwarder.h"
 #include "mozilla/layers/TextureForwarder.h"
 #include "mozilla/layers/TiledContentClient.h"
-
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 
 #include "nsComponentManagerUtils.h"
 
@@ -142,7 +141,7 @@
   }
 
   RefPtr<TextureClient> newClient;
-  if (gfxPrefs::ForceShmemTiles()) {
+  if (StaticPrefs::ForceShmemTiles()) {
     // gfx::BackendType::NONE means use the content backend
     newClient = TextureClient::CreateForRawBufferAccess(
         mSurfaceAllocator, mFormat, mSize, gfx::BackendType::NONE, mBackend,